packages/app/ui/search/SearchBar.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import { Icon } from "../icon/Icon";
+import { twMerge } from "tailwind-merge";
+
+interface SearchBarProps {
+  className?: string;
+  placeholder?: string;
+}
+
+const SearchBar = ({ className, placeholder = "Search" }: SearchBarProps) => {
```

Comment:
I think giving props to a component that's only being used once isn't good. What do you think about creating a concrete SearchBar for the TokenPicker? When we need this component more than once we can clearly think in which props we need :)